sormas-ui/src/main/java/de/symeda/sormas/ui/user/UserRoleEditForm.java

```diff
@@ -0,0 +1,145 @@
+/*
+ * SORMAS® - Surveillance Outbreak Response Management & Analysis System
+ * Copyright © 2016-2022 Helmholtz-Zentrum für Infektionsforschung GmbH (HZI)
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see <https://www.gnu.org/licenses/>.
+ */
+
+package de.symeda.sormas.ui.user;
+
+import static de.symeda.sormas.ui.utils.LayoutUtil.fluidRowLocs;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+import java.util.Set;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+import java.util.stream.Stream;
+
+import com.vaadin.icons.VaadinIcons;
+import com.vaadin.shared.ui.ContentMode;
+import com.vaadin.ui.Label;
+import com.vaadin.v7.ui.CheckBox;
+import com.vaadin.v7.ui.ComboBox;
+import com.vaadin.v7.ui.Field;
+import com.vaadin.v7.ui.TextArea;
+
+import de.symeda.sormas.api.i18n.Captions;
+import de.symeda.sormas.api.i18n.Descriptions;
+import de.symeda.sormas.api.i18n.I18nProperties;
+import de.symeda.sormas.api.user.JurisdictionLevel;
+import de.symeda.sormas.api.user.UserRight;
+import de.symeda.sormas.api.user.UserRoleDto;
+import de.symeda.sormas.api.user.UserRoleReferenceDto;
+import de.symeda.sormas.ui.utils.AbstractEditForm;
+import de.symeda.sormas.ui.utils.CssStyles;
+import de.symeda.sormas.ui.utils.components.CheckboxSet;
+
+public class UserRoleEditForm extends AbstractEditForm<UserRoleDto> {
+
+	private static final long serialVersionUID = 8099247063020818190L;
+
+	private static final String USER_RIGHTS_LABEL_LOC = "userRightsLabel";
+	private static final String TEMPLATE_INFO_LOC = "templateInfo";
+	private static final String TEMPLATE_USER_ROLE = "templateUserRole";
+
+	private final static List<String> defaultRightsOrder = Arrays.asList("_VIEW", "_EDIT", "_CREATE");
+
+	private static final String HTML_LAYOUT = fluidRowLocs(TEMPLATE_INFO_LOC)
+		+ fluidRowLocs(UserRoleDto.CAPTION, TEMPLATE_USER_ROLE)
```

Comment:
I know it's different in the mockup, but it would probably be better to move the template drop-down into the first row of the layout and the caption and jurisdiction level into the second. Basically the same way it's in the create form. This way the template info is much more clear.